internal-services/catalog/iib-pipeline.yaml

```diff
@@ -19,11 +19,9 @@ spec:
     - name: iibServiceConfigSecret
       type: string
       description: Secret containing IIB service Config
-      default: iib-services-config
-    - name: iibFromImageOverwriteCredentials
+    - name: iibOverwriteFromIndexCredential
       type: string
       description: Secret with overwrite FromImage credentials to be passed to IIB
-      default: iib-overwrite-fromimage-credentials
```

Comment:
removing the `default` is desired in this PR?

Reply:
nevermind. I see the reason in the other PR

Reply:
same.